resource-providers/src/main/java/io/opentelemetry/resourceproviders/AppServerServiceNameProvider.java

```diff
@@ -37,11 +38,14 @@ public AppServerServiceNameProvider() {
   @Override
   public Resource createResource(ConfigProperties config) {
     String serviceName = detectServiceName();
-    if (serviceName != null) {
-      logger.log(INFO, "Auto-detected service name '{0}'.", serviceName);
-      return Resource.create(Attributes.of(SERVICE_NAME, serviceName));
+    if (serviceName == null) {
+      logger.log(
+          WARNING,
```

Comment:
`WARNING` and `INFO` are pretty high logging levels -- these messages will be printed out for everyone who's using this module (so, all Splunk distro users)

Reply:
True, but only if it got past `shouldApply()`. In that case, I do think it's helpful to see what is happening...but if you feel strongly about bumping all of the logging down to  `FINE` or something I can do that. 

Personally, I'd rather have more logs for troubleshooting than not enough to speculate....

Reply:
`shouldApply()` also logs a couple `INFO` messages that perhaps should be `FINE`.

I totally get it that it's valuable for troubleshooting, but we've had customers complain about the agent printing out stuff that does not match their logging patterns. E.g. let's assume that somebody runs a simple one jar app and expects the service name to be detected from the file name -- this resource provider would print 4 extra logs that are totally unrelated to their use case (because the jar name detector runs last).

Reply:
`FINE` I'll change it. 🙃